### DISCUSSION TITLE
Where can I find V9 hotkey commands for use in config file?

### DISCUSSION THREAD

**Discussion by gaberiel44** [d2phap/ImageGlass#1485] [Q&A] [answered]
Hello, I am currently using V9 beta 3, I know there is currently no UI to configure settings, its fine, I prefer config files anyways.

But where can I commands for various imageglass functions? or even an example of an imageglass config file that has its 
-` "MenuHotkeys": {}`,
-`"MouseClickActions": {}`
-`"MouseClickActions": {}`
populated.

Thank you.

**Comment by d2phap** [comment, marked as answer]
You can find it in the Details section of the release note.

### `MenuHotkeys` is introduced in v9.0 beta 1, and supports multiple hotkey in v9.0 beta 3:
```
MenuHotkeys: {
  <menu_name>: hotkey[]
}
```
while
- `menu_name` is the name of the menu variable, starts with `Mnu`. You can find the all menu names here https://github.com/d2phap/ImageGlass/blob/019189c8dbf079a199278615a3d27d9a3da09d20/v9/ImageGlass/FrmMain.Designer.cs#L1313, or the pre-defined menu: https://github.com/d2phap/ImageGlass/blob/019189c8dbf079a199278615a3d27d9a3da09d20/v9/ImageGlass/FrmMain/FrmMain.Configs.cs#L40
- `hotkey` is an array of hotkey string, for example: `["Ctrl+A", "Shift+Z"]`


### `MouseClickActions` is introduced in v9.0 beta 2:
You can see the full example here: https://github.com/d2phap/ImageGlass/issues/991#issuecomment-1238130837

**Reply by gaberiel44** [reply]
Wonderfull. Thanks allot for this!
